content/articles/The-concept-of-cookies-in-servlets/index.md

```diff
@@ -0,0 +1,222 @@
+### Introduction
+
+Cookies are text-based bits of information sent by the webserver to the client. It's delivered along with a client's response. A cookie, in simple words, is a method of keeping a client's state by a web server. When a client makes a new request, it sends the request along with all cookies accessible for the server to recognize. When a cookie's life expectancy ends, the client browser deletes it.
+### Prerequisites
+- It's necessary to have a basic understanding of Java and the servlet's introduction part.
+- Install an open-source editor, such as Eclipse, Netbeans, etc. In this article, we will use [Eclipse IDE](https://www.eclipse.org/downloads/packages/release/kepler/sr1/eclipse-ide-java-developers).
+- Your machine should have Tomcat Server installed. It's available for download on their official [website](https://tomcat.apache.org/). We'll use TomCat version 10 in this case.
+
+### Objectives
+Learners should be able to answer the following questions by the end of this article:
+1. What are cookies? And why is it important to use it?
+2. How do cookies work? What types of cookies are there in Java Servlet?
+3. What is Cooke's class? And what are the methods required for cookies?
+4. How to implement cookies in the Java Servlet program?
+
+### Table of content
+- [API for Cookies](#api-for-cookies)
+- [How does Cookie work](#how-does-cookie-work)
+- [Types of Cookies in Java Servlet](#types-of-cookies-in-java-servlet)
+    - [Non persistent cookies](#non-persistent-cookies)
+    - [persistent cookies](#persistent-cookies)
+- [Advantage of Cookies](#advantage-of-cookies)
+- [Disadvantage](#disadvantage)
+- [Cookie class and the methods required for cookies](#cookie-class-and-the-methods-required-for-cookies)
+   - [Cookie class](#cookie-class)
+   - [Methods of Cookie class](#methods-of-cookie-class)
+- [How to create, delete, and get cookies](#how-to-create-delete-and-get-cookies)
+  - [Creating a new Cookie object](#creating-a-new-cookie-object)
+  - [Setting up lifespan for a cookie](#setting-up-lifespan-for-a-cookie)
+  - [Sending the cookies to the client](#sending-the-cookies-to-the-client)
+  - [Getting cookie from client request](#getting-cookie-from-client-request)
+  - [Deletting cookies](#deletting-cookies)
+- [Example to demonstrate Servlet Cookie](#example-to-demonstrate-servlet-cookie)
+- [Conclusion](#conclusion)
+
+### API for Cookies
+
+The `Cookie` class in the Servlet API is used to create cookies. Cookies are created using the Servlet API's 'Cookie' class. This class employs a variety of approaches to give various cookie activities. The `addCookie()` method, for example, is used to add cookies to the response object. It sends cookie data from a client to a server or from a server to a client using the `HTTP` response. The `getCookies()` method, on the other hand, is used to access the cookies that have been added to the response object.

```

Comment:
Change "It sends cookie data from a client to a server or from a server to a client using the `HTTP` response." to "Then, it sends cookie data from a client to a server or server to a client using the `HTTP` response."